scripts/tests/secrets-integration.sh

```diff
@@ -14,9 +14,9 @@ test -n "$EARTHLY_TOKEN"
 "$earthly" account login
 
 # fetch shared secret key (this step assumes your personal user has access to the /earthly-technologies/ secrets org
-ID_RSA=$($earthly secrets get -n /earthly-technologies/github/other-service+github-cinnamon@earthly.dev/id_rsa)
+ID_RSA=$($earthly secrets get -n /earthly-technologies/earthly-secrets/manitou/id_rsa)
 
-# now that we grabbed the cinnamonthecat credentials, unset our token, to ensure that we're only testing using cinnamonthecat's credentials
+# now that we grabbed the manitou credentials, unset our token, to ensure that we're only testing using manitou's credentials
```

Comment:
just curious what the inspiration behind the new manitou name is?

Reply:
I had some random music on which reminded me of the movie; there's also a very friendly black lab that lives a few doors down named manitou.